website/docs/r/spring_cloud_application_insights_application_performance_monitoring.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+subcategory: "Spring Cloud"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud_application_insights_application_performance_monitoring"
+description: |-
+  Manages a Spring Cloud Application Insights Application Performance Monitoring.
+---
+
+# azurerm_spring_cloud_application_insights_application_performance_monitoring
+
+-> **NOTE:** This resource is applicable only for Spring Cloud Service with enterprise tier.
```

Comment:
change to "This resource is only applicable for Spring Cloud Service enterprise tier"